tests/generate_fixtures/generate_target_dnadiff_files.py

```diff
@@ -48,7 +48,6 @@
 
 from pyani_plus.tools import (
     get_delta_filter,
-    get_dnadiff,
```

Comment:
Is there a reason why `pre-commit-ci[bot]` doesn't like this import and keeps removing it?

Reply:
It looks like an unused import, so it should be removed.

Reply:
Ah - I see it now, I messed up the merge conflict and dropped the line using it...